extensions/geode-modules-tomcat8/src/distributedTest/java/org/apache/geode/modules/session/Tomcat8SessionsClientServerDUnitTest.java

```diff
@@ -80,6 +82,7 @@ public void setupServer() throws Exception {
     sessionManager = manager;
     sessionManager.setEnableCommitValve(true);
     server.getRootContext().setManager(sessionManager);
+    AuthConfigFactory.setFactory(null);
```

Comment:
Just curious, what is this line doing?

Reply:
the newer version of tomcat8 is expecting some sort of auth module being configured in web.xml if we don't do this.

Reply:
It's only in the test that we are disabling 3rd party authentication